modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit labels and  annotations for a standalone virtual machine instance (VMI) using the web console. Other items displayed in the *Overview* page for a standalone VMI are not editable.
+
+.Procedure
+
+. Click *Workloads -> Virtual Machines*  from the side menu. A list of virtual machines (VMs) and standalone VMIs display.
+
+. Select a standalone VMI to open the *Virtual Machine Instance Overview* screen.
```

Comment:
There are no editable items on `Overview` page, you dropped the step to navigate to `Details` page.

Reply:
I did a bluejeans session with @matthewcarleton and together we reviewed the user interface (he has access to the current user interface and cnv 2.4 lab environment). I have updated this procedure to show that user has to click on the Details tab to be able to edit the annotations. That is the only action that a user can perform with respect to editing a VMI from the Details page. Please review.

Reply:
Looks good.